packages/dx-vue-grid-bootstrap4/src/templates/paging-panel/pagination.js

```diff
@@ -0,0 +1,203 @@
+import { firstRowOnPage, lastRowOnPage, calculateStartPage } from '@devexpress/dx-grid-core';
+
+const renderPageButtons = (
```

Comment:
Maybe its better to convert this method to a component?

Reply:
No, because Vue can't return a markup array.

Reply:
You can use functional component. It allows to return multiple root nodes